### ISSUE TITLE
Add context menu in project explorer

### ISSUE THREAD

**Issue by acababaev** [denis-shienkov/vscode-qbs#62]
could be good, if qbs project explorer would handle context menu, or handle key bindings (for example F2 - rename), just like standard Explorer of vscode

**Comment by denis-shienkov** [comment]
Agreed, but seems, it is hard to do (I can't find any examples for this)... AFAIK, this feature is not implemented even in the CMake Tools extension.

**Comment by acababaev** [comment]
ok, I'm just comparing with qt creator. I thought to switch to vs code, but for C++, Qt, Qml, Qml, qt creator is the best development environment, there are a lot of features in vscode, including in the form of plugins
